TehraniFarzam/HW5/05.1_PRACTICAL_TASKS-1 .py

```diff
@@ -0,0 +1,8 @@
+list_of_integers = [1, 3, 4, 8, 14, 75, 99]
+list_of_floats = []
+
+for integer in list_of_integers:
```

Comment:
for integer in list_of_integers:
    list_of_floats.append(float(integer))

Reply:
The suggested changes were modified in line 5